### PR TITLE
Feature/auth

### PR DESCRIPTION
**Pull request by pgallardor**
Authentication utils for Explorer and Hunter

### CODE REVIEW

File: packages/component-library/src/utils/auth/credentials.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+) {
+  const config = {
+    baseUrl,
+    withCredentials: true,
+    headers: {
+      "Content-Type": "application/json",
+    },
+  };
+
+  console.log("credentials baseurl", baseUrl);
```
Comment (dirodriguezm) [inline packages/component-library/src/utils/auth/credentials.ts:19]: keep in mind deleting this at some point

Comment (pgallardor) [reply packages/component-library/src/utils/auth/credentials.ts:19]: yessir

File: packages/component-library/src/utils/stores.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    async function credentialsLogin(username: string, password: string) {
+      try {
+        await _credentials.signIn(username, password);
+        loggedUser.value = {
+          isLogged: true,
+          user: username,
+        };
+      } catch (e) {
+        console.error("Invalid username or password");
```
Comment (dirodriguezm) [inline packages/component-library/src/utils/stores.ts:38]: Shouldn't we just throw errors at this point so the application can handle them ?

File: packages/hunter/package.json [hunk 3]
```diff
@@ -15,6 +15,7 @@
     "@alercebroker/http-client": "^1.1.0-rc.16",
     "@quasar/extras": "^1.15.0",
     "@vue/test-utils": "^2.0.2",
+    "axios": "^0.27.2",
```
Comment (dirodriguezm) [inline packages/hunter/package.json:18]: I guess this shouldn't be a direct dependency here ?

File: packages/component-library/src/utils/auth/oauth.ts [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        ...config,
+        headers: {
+          "Content-Type": "application/x-www-form-urlencoded",
+        },
+      });
+      const { access, refresh } = session;
+      tokenHandler.storeToken({ access, refresh });
+    } catch (e) {
+      throw e;
+    }
```
Comment (dirodriguezm) [inline packages/component-library/src/utils/auth/oauth.ts:33]: image

Comment (pgallardor) [reply packages/component-library/src/utils/auth/oauth.ts:33]: lmao

File: packages/hunter/package.json (hunk 3 above)
Comment (pgallardor) [reply packages/hunter/package.json:18]: nope, I installed it just for testing purposes

File: packages/component-library/src/utils/stores.ts (hunk 2 above)
Comment (pgallardor) [reply packages/component-library/src/utils/stores.ts:38]: true

**Review by dirodriguezm — APPROVED** [review]

**Pull request merged**